### PR TITLE
airbnb search bar

### PR DESCRIPTION
**Pull request by NikitaKarbazin**
- [DEMO_LINK](https://NikitaKarbazin.github.io/layout_search-bar-airbnb)
- [TEST_REPORT_LINK](https://NikitaKarbazin.github.io/layout_search-bar-airbnb/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,16 +1,46 @@
...
+    <link rel="stylesheet" href="./style.css"
+    >
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form action="meta.acedemy" class="wraper-big"
+       data-qa="big" method="post"
+    >
```
Comment (mykola-koval) [inline src/index.html:19]:
```suggestion
    <form 
      action="meta.acedemy"
      method="post"
      class="wraper-big"
      data-qa="big"
    >
```

File: src/style.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
+@font-face {
+  font-family: AvenirBook;
+  src: url(fonts/Avenir-Book.ttf) ;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: AvenirBold;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: 300;
```
Comment (mykola-koval) [inline src/style.css:10]: use the same font-family name for these fonts, but different font-weight

File: src/style.css [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+  line-height: 19px;
+  font-size: 14px;
+  color: #3d4e61;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.big:focus-within {
```
Comment (mykola-koval) [inline src/style.css:77]: you don't have any descendants of this element so you can just use :focus pseudoclass

File: src/style.css [hunk 4]
```diff
@@ -0,0 +1,109 @@
...
+.big {
+  box-sizing: border-box;
+  height: 70px;
+  width: 100%;
+  font-family: AvenirBold, AvenirBook, sans-serif;
+  padding-left: 62px;
+  line-height: 22px;
+  font-size: 16px;
+  color: #3d4e61;
+  background: #fff;
```
Comment (mykola-koval) [inline src/style.css:56]: background is a shorthand property, use background-color 
https://developer.mozilla.org/en-US/docs/Web/CSS/background

**Review by mykola-koval — CHANGES_REQUESTED** [review]
text in the searchbars should only become bold on focus

File: src/index.html [hunk 5]
```diff
@@ -1,16 +1,55 @@
...
     <title>Search bar airbnb</title>
-    <link rel="stylesheet" href="./style.css">
+    <link rel="stylesheet" href="./style.css"
+    >
   </head>
+
   <body>
-    <h1>Search bar airbnb</h1>
+
+    <form
```
Comment (Olexandr6) [inline src/index.html:17]:
```suggestion
    <form
```

File: src/index.html [hunk 6]
```diff
@@ -1,16 +1,55 @@
...
+      <label for="search-big">
+        <img
+          src="images/Search.svg"
+          alt="airbnb"
+          class="icon-big"
+        >
+        <input type="search" class="big"
+          placeholder="Try &#34Los Angeles&#34" tabindex="1"
+          data-qa="keypress" id="search-big"
+        >
```
Comment (Olexandr6) [inline src/index.html:32]: [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 5th point

File: src/index.html [hunk 7]
```diff
@@ -1,16 +1,55 @@
...
+      method="post"
+    >
+      <label for="search-small">
+        <img src="images/Search.svg" alt="airbnb"
+          class="icon-small"
+        >
+        <input type="search" class="small"
+          placeholder="Try &#34Los Angeles&#34" tabindex="1"
+          id="search-small"
+        >
```
Comment (Olexandr6) [inline src/index.html:49]: The same problem. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 5th point

File: src/style.css [hunk 8]
```diff
@@ -0,0 +1,113 @@
...
+  margin-bottom: 20px;
+  position: relative;
+  margin-top: 20px;
+}
+
+.wraper-small {
+  position: relative;
+}
+
+input[type="search"]::-webkit-search-cancel-button {
```
Comment (Olexandr6) [inline src/style.css:27]: Why do need this here?

File: src/style.css [hunk 9]
```diff
@@ -0,0 +1,113 @@
...
+  font-size: 16px;
+  line-height: 22px;
+}
+
+.small::placeholder {
+  font-family: Avenir, sans-serif;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 19px;
+}
```
Comment (Olexandr6) [inline src/style.css:113]: You duplicate a bunch of styles. [Checklist](https://github.com/mate-academy/layout_search-bar-airbnb/blob/master/checklist.md) the 2nd point

**Review by Olexandr6 — CHANGES_REQUESTED** [review]

File: src/style.css (hunk 9 above)
Comment (NikitaKarbazin) [reply src/style.css]: I was really upset when try to write code css, and one aim was make solution. And when it was completed i click pull requst and all. in future i be better

**Review by Olexandr6 — APPROVED** [review]
Cool🔥

**Pull request closed without merging**